modules/ROOT/pages/6.8.2-release-notes.adoc

```diff
@@ -0,0 +1,89 @@
+
+As of {productname} 6.8.2, the name of the `align` bespoke select button is changed to “Alignment”, followed by the currently selected value. This results in the accessible labels being “Alignment left”, “Alignment right”, “Alignment center”, or “Alignment justify” depending on the value selected, which more clearly indicates the currently selected text alignment.
+
+=== Accessible names of bespoke select toolbar buttons including `align`, `fontfamily`, `fontsize`, `blocks`, and `styles` were incorrectly translated.
+// #TINY-10426 #TINY-10435
+
+As of {productname} 6.8.1, the accessible labels of the bespoke select toolbar buttons announced both the name of the button and the currently selected value, for example “Block Paragraph”. However, corresponding translation strings with placeholders were not added. This resulted in the current value being translated as expected, but not the name of the button. 
+
+In {productname} 6.8.2, new translation strings were added to ensure names of the bespoke select toolbar buttons are translated as well, making their accessible labels appropriately internationalized.
+
+=== Clicking inside table cells with heavily nested content could cause the browser to hang.
+// #TINY-10380
+Previously when clicking below content that is heavily nested inside a block such as a table cell, would cause the browser to hang.
+
+{productname} 6.8.2 addresses this, by optimizing algorithm that was causing the browser to hang.
+
+As a result, the browser no longer hangs, when clicking inside table cells with heavily nested content.
+
+=== Toggling a list that contains an LI element having another list as its first child would remove the remaining content within that LI element.
+// #TINY-10414
+In a previous xref:6.7.2-release-notes.adoc#toggling-a-list-that-contained-a-list-item-element-li-which-in-turn-contained-another-list-item-element-as-its-first-child-removed-other-content-within-the-first-list-item-element[bugfix] for lists, some element structures would not recreate the `<li>` tags when a list element was toggled using the `bullist` button in the toolbar.
+
+.Example structure 
+[source, html]
+----
+<li>
+  <ul>...</ul>
+  <something-else/>
+</li>
+----
+
+As a consequence, any tags nested within the `<li>` would be deleted from the {productname} document, resulting in content loss for the user.
+
+{productname} 6.8.2 addresses this by improving the representation of the list to manage this case.
+
+As a result, now, when toggling `<li>` elements using the `bullist` button, content within a list is now correctly recreated the content is retained as expected.
```

Comment:
@lorenzo-pomili waiting for review on this section.

Reply:
approved